### PR TITLE
MF-116 Packmap gets wrong path for esm-error-handling and esm-root-config

### PR DESCRIPTION
**Pull request by brandones**
https://issues.openmrs.org/browse/MF-116

Changes the package main from the .lib.js to .defaults.js.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -2,12 +2,13 @@
   "name": "@openmrs/esm-root-config",
   "version": "1.1.2",
   "description": "A javascript module for setting up and configuring your SPA ",
-  "main": "dist/openmrs-esm-root-config.lib.js",
+  "main": "dist/openmrs-esm-root-config.defaults.js",
```
Comment (fatmali) [inline package.json]: `openmrs-esm-root-config.lib.js` exports a number of helper functions to be used in distro root configs. When an organization needs to add custom apps to their distro, they can extend the `openmrs-esm-root-config` using some of the functions provided in that file. Removing it from `main` means they can't be imported and used. @joeldenning   is this right?

Comment (joeldenning) [reply package.json]: That is right.

esm-root-config is a weird one for packmap -- it should actually point to the distro-specific root config that in turn has the `@openmrs/esm-root-config` package as a dependency.

Comment (brandones) [reply package.json]: Got it. So should we move `openmrs-esm-root-config.defaults.js` into its own module?

Comment (joeldenning) [reply package.json]: I don't think so

Comment (brandones) [reply package.json]: I think we should have some way for an implementation to get SPA running on their server without writing their own root config override

Comment (brandones) [reply package.json]: Oh yeah, I guess that's easy enough to solve just by editing the import map after generation. Closing.

**Pull request closed without merging**